scripts/registry-census/verify_detection.py

```
@@ -0,0 +1,138 @@
+import os
+import sys
+
+EXPECTED_PACKS = 7
+
+
+def main() -> int:
+    out = os.environ.get('MATRIX_OUT', '/tmp/matrix')
+    rows: dict[str, dict] = {}
+    for name in os.listdir(out) if os.path.isdir(out) else []:
+        if name.endswith('.json') and not name.startswith('_'):
+            r = json.load(open(os.path.join(out, name)))
```

Ruff (0.16.1)

[warning] Use a context manager for opening files

(SIM115)